TFT/src/User/API/Language/language_hu.h

```diff
@@ -282,18 +282,18 @@
     #define STRING_LOAD_STARTED           "A filament betöltése folyamatban van,\nnem lehetséges további művelet,\nmíg véget nem ér.\nKérlek várj!"
     #define STRING_UNLOAD_STARTED         "A filament kiadása folyamatban van,\nnem lehetséges további művelet,\nmíg véget nem ér.\nKérlek várj!"
     #define STRING_HEATERS_ON             "Néhány fűtőelem még üzemel.\nKikapcsolod a fűtőelemeket?"
-    #define STRING_PRINT_FINISHED         "   Printing\n   finished."  // keep it short and in this format!
-    #define STRING_MAIN_SCREEN            "Main"
+    #define STRING_PRINT_FINISHED         "   Nyomtatás\n   elkészült."  // keep it short and in this format!
+    #define STRING_MAIN_SCREEN            "Menü"
```

Comment:
It is a label for a button that doesn't do the same as "Menu" that is on the main page. It should be something like: "Főoldal", "Kezdőlap", "Kezdet", "Alap", etc

Reply:
That's what I was thinking. But I couldn't connect to a screen. He didn't show up anywhere for me. But I'll change it then.

Reply:
It shows up when a print is finished. The Pause/Resume button changes to a new one that takes you to the startup page.

Reply:
I printed today. But I didn't notice in the end. I'll check again.

Reply:
Are you on the latest merge? It was introduced in the second to last merge.

Reply:
Yes. I downloaded it today. But maybe I just didn't notice.

Reply:
Thanks! I will check it tomorrow.

Reply:
Got it. Thank you "G28". I just checked. Returns to the main screen. So grammatically "Főoldal" or "Kezdőlap" is correct. There was an inconsistent translation of "Confirm" that was "Igen." Well, I corrected that to "Oké." Because there is one that suits you better. However, since a question, sentence, statement also occurs, it is quite difficult to find a word that fits every situation.